src/Widgets/DataDeleteButtonWidget/DataDeleteButtonWidgetComponent.tsx

```diff
@@ -27,15 +33,15 @@ provideComponent(DataDeleteButtonWidget, ({ widget }) => {
             content={widget}
             attribute="cancelTitle"
             tag="button"
-            className="btn btn-sm"
+            className={buttonClassNames.join(' ')}
             onClick={onDeleteRejected}
           />
 
           <ContentTag
             content={widget}
             attribute="confirmTitle"
             tag="button"
-            className={`btn btn-sm ${buttonColor}`}
+            className={`${buttonClassNames.join(' ')} ${buttonColor}`}
```

Comment:
* [ ] FYI seeing both string interpolation and string join used at the same time to interpolate a class name looks weird.